userguide: add ctrl alias

lasers opened the pull request.

stapelberg commented on docs/userguide:
```diff
@@ -1,4 +1,4 @@
-i3 User’s Guide
+/i3 User’s Guide
```
Please remove this accidental modification.

stapelberg commented on docs/userguide:
```diff
@@ -392,9 +392,12 @@ bindsym --release $mod+x exec --no-startup-id import /tmp/latest-screenshot.png
 Standard modifiers, see +xmodmap(1)+
 
+Ctrl::
+An alias for +Control+.
```
Meta: sorry for the late reply, I didn’t get an email when you pushed your changes (please comment in the future).

Can you move this into the previous paragraph? Since Control is documented there and Ctrl is just an alias, I think it makes more sense if it went into the previous paragraph.

lasers commented:
Do you want just `Mod1, Mod2, Mod3, Mod4, Mod5, Shift, Control, Ctrl` ? It looks odd to me putting Ctrl right next to Control because they both are same thing and is not one of the standard modifiers on `xmodmap` (mentioned on the next line). I tried `Control (Ctrl)` and it does not look that nice. I wanted to find a way to make it clear that it's an alias (without making it too ugly). It could help if people knew that Ctrl is an alias and i3wm developers might like for people to stick with real Control modifier name. If Ctrl is mostly likely the only alias we will have in the long run, then I will go ahead and do `Control, Ctrl` although it may look slightly weird? Otherwise, it could be a good place to put in for future if you wish to introduce some other new aliases such as Alt and Win.

stapelberg commented:
We could just mention the fact that Ctrl is an accepted alias for Control in the text paragraph itself. See https://github.com/i3/i3/pull/1857/files where I point out that Mode_switch is an alias for Group2.

lasers commented:
Fixed and squashed excessive commits. What should I do about the other commits not made by me on the list since I had this PR left open for a while? Please review. I will comment after any fixes in the future. Thanks.

stapelberg commented:
You can use `git rebase next` to make your commit be applied to an up-to-date `next` branch without any other commits.

Airblader commented:
We have adapted the default config by now and there's a dedicated issue ( #1855 ) for the tray part of this PR. It should be removed.

lasers commented:
Removing / Closing.
